packages/angular-cli/models/webpack-build-test.js

```diff
@@ -66,7 +67,7 @@ const getWebpackTestConfig = function (projectRoot, environment, appConfig) {
         { test: /\.(jpg|png)$/, loader: 'url-loader?limit=128000' },
         { test: /\.html$/, loader: 'raw-loader', exclude: [path.resolve(appRoot, appConfig.index)] }
       ],
-      postLoaders: [
+      postLoaders: debug ? [] : [
```

Comment:
ok @filipesilva . i will update this branch to the latest changes in master and fix the linting issue you mentioned.